sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -400,6 +400,10 @@ case class AlterTableDropPartition(
       throw new AnalysisException(
         "ALTER TABLE DROP PARTITIONS is not allowed for tables defined using the datasource API")
     }
+    if (specs.size > 1) {
+      throw new AnalysisException(
+        s"Alter Table Drop Partition is not allowed to drop more than one partitions.")
```

Comment:
actually I don't think we should throw an exception here. Let's move the discussion to the main thread.